lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -470,6 +477,11 @@ def parse_search_string(search_string)
               next if search_term.length == 0
               keyword.downcase!
               search_term.downcase!
+
+              if keyword == "type"
+                search_term = MODULE_TYPE_SHORTHANDS[search_term] if MODULE_TYPE_SHORTHANDS.key?(search_term)
```

Comment:
This might be cleaner, or harder to read:

`search_term = MODULE_TYPE_SHORTHANDS.fetch(search_term, search_term)`

Docs for fetch: https://apidock.com/ruby/Hash/fetch

Reply:
It's not immediately clear with `fetch` what's happening, `.key?` may be wordier but it's easier to understand IMO